workers/unity/Packages/com.improbable.gdk.deploymentlauncher/DeploymentConfigurations.cs

```diff
@@ -365,23 +365,52 @@ public class DeploymentInfo
         /// <summary>
         ///     The SpatialOS project that the deployment is running in.
         /// </summary>
-        public string ProjectName { get; }
+        public string ProjectName { get; private set; }
 
         /// <summary>
         ///     The name of the deployment.
         /// </summary>
-        public string Name { get; }
+        public string Name { get; private set; }
 
         /// <summary>
         ///     The id of the deployment.
         /// </summary>
-        public string Id { get; }
+        public string Id { get; private set; }
 
-        public DeploymentInfo(string projectName, string name, string id)
+        /// <summary>
+        ///     The start time of the deployment.
+        /// </summary>
+        public string StartTime { get; private set; }
```

Comment:
Why a string?

Reply:
Its passed over JSON, so we do the `DateTime` -> `string` conversion in the `.DeploymentLauncher` and bubble it back up.

The alternative is to pass it up as a unix timestamp and then decode it in the Editor, but it doesn't make a ton of difference imo?

Reply:
Actually it gives the caller of the `.DeploymentLauncher` more flexibility in how they use the timestamp. Will change to that